doc/manual/en/colab/index.rst

```diff
@@ -0,0 +1,435 @@
+    (``git clone git@github.com:gh-user/CometVisu.git``). If cloned
+    over HTTPS (``https://github.com/CometVisu/CometVisu.git``) the
+    URLs will look different.
+
+Abstract
+""""""""
+
+.. code-block:: bash
+
+    # query current settings
+    git remote -v
+    >>> origin	git@github.com:gh-user/CometVisu.git (fetch)
+    >>> origin	git@github.com:gh-user/CometVisu.git (push)
+
+    # Get updates from the original repository
+    git remote set-url origin git@github.com:CometVisu/CometVisu.git
+
+    # check current settings
+    git remote -v
+    >>> origin	git@github.com:CometVisu/CometVisu.git (fetch)
+    >>> origin	git@github.com:gh-user/CometVisu.git (push)
+
+    # synchronize develop branch with original repository
+    git checkout develop
+    git pull
+
+Detailed approach
+"""""""""""""""""
+
+The current settings can be queried with the command ``git remote -v``.
+Usually, this returns the following values:
+
+.. code-block:: bash
+
+    origin	git@github.com:gh-user/CometVisu.git (fetch)
+    origin	git@github.com:gh-user/CometVisu.git (push)
+
+This means that when updating (*fetch*,*pull*) and also
+*pushing* git uses the user's fork on github.
+But you want to load changes to the main repository into
+the local clone as easily as possible. To do this, change the
+*fetch* URL of the remotes with the following command:
+``git remote set-url origin git@github.com:CometVisu/CometVisu.git``.
+After that, ``git remote -v`` should deliver the following output:
+
+.. code-block:: bash
+
+    origin	git@github.com:CometVisu/CometVisu.git (fetch)
+    origin	git@github.com:gh-user/CometVisu.git (push)
+
+This fulfills the requirements and you can create a local *develop*
+branch (this contains the latest developments) simply by
+a ``git pull`` update.
+
+.. HINT::
+   This procedure is only advisable if you only make changes to the
+   CometVisu on one computer. If you use several computers at the
+   same time, you will inevitably have to make the changes you have
+   from the other computer in your own fork *pushed* got into the
+   local clone. For this you can set up a second remote with
+   ``git remote add fork git@github.com:gh-user/CometVisu.git`` and then *pull*
+   ``git pull fork develop``
+
+.. TODO::
+
+    * other useful git commands (branches delete, status, ...)
+    * merge-conflicts
+    * Flowchart for workflow change -> pull request
+
+Help with the documentation
+---------------------------
+
+The CometVisu documentation is produced using the
+`Sphinx - Python Documentation Generator <http://www.sphinx-doc.org/>`__
+generated. The documentation is therefore written in the reStructuredText
+format (RST for short). This is a simplified markup language that is easy
+to read in plain text form and has the advantage of being simple
+to be converted to other formats [RST-Wiki]_.
+Zum tieferen Einstieg in die Syntax des RST-Formats sind folgende
+Links geeignet:
```

Comment:
For more detailed information about the syntax of the RST format please consult those links: